GameServer/Logic/GameObjects/AzirTurret.cs

```diff
@@ -0,0 +1,209 @@
+                return;
+            }
+
+            stats.HealthPoints.BaseValue = inibin.getFloatValue("Data", "BaseHP");
+            stats.CurrentHealth = stats.HealthPoints.Total;
+            stats.ManaPoints.BaseValue = inibin.getFloatValue("Data", "BaseMP");
+            stats.CurrentMana = stats.ManaPoints.Total;
+            stats.AttackDamage.BaseValue = inibin.getFloatValue("DATA", "BaseDamage");
+            stats.Range.BaseValue = TURRET_RANGE;//inibin.getFloatValue("DATA", "AttackRange");
+            stats.MoveSpeed.BaseValue = inibin.getFloatValue("DATA", "MoveSpeed");
+            stats.Armor.BaseValue = inibin.getFloatValue("DATA", "Armor");
+            stats.MagicResist.BaseValue = inibin.getFloatValue("DATA", "SpellBlock");
+            stats.HealthRegeneration.BaseValue = inibin.getFloatValue("DATA", "BaseStaticHPRegen");
+            stats.ManaRegeneration.BaseValue = inibin.getFloatValue("DATA", "BaseStaticMPRegen");
+            stats.AttackSpeedFlat = 0.625f / (1 + inibin.getFloatValue("DATA", "AttackDelayOffsetPercent"));
+
+            stats.HealthPerLevel = inibin.getFloatValue("DATA", "HPPerLevel");
+            stats.ManaPerLevel = inibin.getFloatValue("DATA", "MPPerLevel");
+            stats.AdPerLevel = inibin.getFloatValue("DATA", "DamagePerLevel");
+            stats.ArmorPerLevel = inibin.getFloatValue("DATA", "ArmorPerLevel");
+            stats.MagicResistPerLevel = inibin.getFloatValue("DATA", "SpellBlockPerLevel");
+            stats.HealthRegenerationPerLevel = inibin.getFloatValue("DATA", "HPRegenPerLevel");
+            stats.ManaRegenerationPerLevel = inibin.getFloatValue("DATA", "MPRegenPerLevel");
+            stats.GrowthAttackSpeed = inibin.getFloatValue("DATA", "AttackSpeedPerLevel");
+
+            setMelee(inibin.getBoolValue("DATA", "IsMelee"));
+            setCollisionRadius(inibin.getIntValue("DATA", "PathfindingCollisionRadius"));
+
+            Inibin autoAttack;
+            if (!RAFManager.getInstance().readInibin("DATA/Characters/" + model + "/" + model + "BasicAttack.inibin", out autoAttack))
+            {
+                if (!RAFManager.getInstance().readInibin("DATA/Characters/" + model + "/Spells/" + model + "BasicAttack.inibin", out autoAttack))
+                {
+                    if (!RAFManager.getInstance().readInibin("DATA/Spells/" + model + "BasicAttack.inibin", out autoAttack))
+                    {
+                        Logger.LogCoreError("Couldn't find turret auto-attack data for " + model);
+                        return;
+                    }
+                }
+            }
+
+            autoAttackDelay = autoAttack.getFloatValue("SpellData", "castFrame") / 30.0f;
+            autoAttackProjectileSpeed = autoAttack.getFloatValue("SpellData", "MissileSpeed");
+        }
+
+        public string getName()
+        {
+            return _name;
+        }
+
+        public Unit GetOwner()
```

Comment:
Use C# style getter/setter, so define this as
```CS
public Unit Owner { get { return _owner; }}
```
or
```CS
public Unit Owner { get; private set; }
```
latter option won't need you to define `_owner` separately, thus recommended if you can use it.